src/app/loginComponents/requests/requests.component.html

```diff
@@ -26,11 +26,22 @@ <h3>Curator Requests</h3>
               on {{ org.dbCreateDate | date }}
             </p>
           </mat-card-subtitle>
+          <app-deregister-modal
+            [id]="'delete-pending-org' + i"
+            class="ml-auto h-50"
+            [toolTipMessage]="'Delete this pending organization: ' + org.displayName"
+            [option]="'organization'"
+            [organizationName]="org.displayName"
+            [organizationID]="org.id"
+            [organizationStatus]="'pending'"
+            [confirmationDialogData]="getDeregisterDialogData(org.displayName, 'pending')"
```

Comment:
this gets executed a bajillion times

Reply:
No longer using deregister-modal, so the properties are deleted.